artifactory/utils/yarn/versionVerify.go

```diff
@@ -0,0 +1,39 @@
+package yarn
+
+import (
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/gofrog/version"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"strings"
+)
+
+const UNSUPPORTED_YARN_VERSION = "4.0.0"
+
+func VerifyYarnVersionSupport(executablePath string) error {
+	versionGetCmdConfig := getVersionCmdConfig(executablePath)
+	output, err := gofrogcmd.RunCmdOutput(versionGetCmdConfig)
+	if err != nil {
+		return errorutils.CheckError(err)
+	}
+	yarnVersionStr := strings.TrimSpace(output)
+	return IsVersionSupported(yarnVersionStr)
+}
+
+func IsVersionSupported(versionStr string) error {
+	yarnVersion := version.NewVersion(versionStr)
+	if yarnVersion.Compare(UNSUPPORTED_YARN_VERSION) < 0 {
```

Comment:
This should check 4.0.0 as well
```
yarnVersion := version.NewVersion(versionStr)
if yarnVersion.Compare(UNSUPPORTED_YARN_VERSION) <= 0 {

}
```
otherwise .Compare returns +1 when yarnversion is 3.x.x and UNSUPPORTED_YARN_VERSION is 4.0.0